ompi/mca/coll/han/coll_han_component.c

```diff
@@ -147,6 +177,37 @@ const char* mca_coll_han_topo_lvl_to_str(TOPO_LVL_T topo_lvl)
     }
 }
 
+static int
+mca_coll_han_query_module_from_mca(mca_base_component_t* c,
+                                   const char* param_name,
+                                   const char* param_doc,
+                                   int info_level,
+                                   uint32_t* module_id,
+                                   char** storage)
+{
+    char *module_name, *endptr = NULL;
+
+    int mod_id = sizeof(available_components) / sizeof(ompi_coll_han_components);
```

Comment:
@bosilca Could we change this to:
```suggestion
    int mod_id = COMPONENTS_COUNT;
```
I ask because `COMPONENTS_COUNT` is used elsewhere in Han, and makes this code a bit more readable.  Indeed, the computation that is here confused me for a while, because it's a `sizeof()` divided by a `sizeof()`, and they're both plural names.  It took a while before I looked up `ompi_coll_han_components` and realized that it was actually a singular component.